src/test/java/RemoveElementsTests.java

```diff
@@ -39,25 +42,25 @@ public void removeElements() {
         var addElementBtn = driver.findElement(addElementBtnLocator);
 
         log.info("Haciendo click 10 veces el en btn add element");
-        int i;
+        var i=0;
         for (i=0; i<10 ; i++) {
```

Comment:
no, usa var en el mismo for, o sea for(var i = 0)

Reply:
done